internal/controller/config/templates/catalog/catalog-configmap.yaml.tmpl

```diff
@@ -19,4 +19,4 @@ data:
         id: default_catalog
         type: yaml
         properties:
-          yamlCatalogPath: /default/default-catalog.yaml
+          yamlCatalogPath: /shared-data/default-catalog.yaml
```

Comment:
@pboyd should this path be changed? or should the init container copy from it's own volume to the default catalog location?

Reply:
I was able to solve the problem by docker / podman pulling the image 
with an explicit architecture command. 
Seems like it is multi-achitecture and my mirroring system was ARM-based